CollapseLauncher/Classes/Helper/Database/DBHandler.cs

```diff
@@ -10,355 +11,412 @@
+            _enabled = c;
+            return c;
+        }
+        set
+        {
+            _enabled           = value;
+            DbConfig.DbEnabled = value;
 
-                _isFirstInit = true; // Force first init
-                if (!value) Dispose(); // Dispose instance if user disabled database function globally
-            }
+            _isFirstInit = true; // Force first init
+            if (!value) Dispose(); // Dispose instance if user disabled database function globally
         }
+    }
         
         
-        private static string? _uri;
-        public static string Uri
+    private static string? _uri;
+    public static string Uri
+    {
+        get
         {
-            get
-            {
-                if (!string.IsNullOrEmpty(_uri)) return _uri;
-                var c = DbConfig.DbUrl;
-                _uri = c;
-                return c;
-            }
-            set
-            {
-                if (value != _uri) _isFirstInit = true; // Force first init if value changed
+            if (!string.IsNullOrEmpty(_uri)) return _uri;
+            var c = DbConfig.DbUrl;
+            _uri = c;
+            return c;
+        }
+        set
+        {
+            if (value != _uri) _isFirstInit = true; // Force first init if value changed
                 
-                _uri           = value;
-                DbConfig.DbUrl = value;
-                _isFirstInit   = true;
-            }
+            _uri           = value;
+            DbConfig.DbUrl = value;
+            _isFirstInit   = true;
         }
+    }
         
-        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        private static string? _token;
+    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+    private static string? _token;
         
-        [DebuggerHidden]
-        [DebuggerBrowsable(DebuggerBrowsableState.Never)]
-        public static string Token
+    [DebuggerHidden]
+    [DebuggerBrowsable(DebuggerBrowsableState.Never)]
+    public static string Token
+    {
+        get
         {
-            get
-            {
-                if (!string.IsNullOrEmpty(_token)) return _token;
-                var c = DbConfig.DbToken;
-                _token = c;
-                return c;
-            }
-            set
-            {
-                if (value != _token) _isFirstInit = true; // Force first init if value changed
+            if (!string.IsNullOrEmpty(_token)) return _token;
+            var c = DbConfig.DbToken;
+            _token = c;
+            return c;
+        }
+        set
+        {
+            if (value != _token) _isFirstInit = true; // Force first init if value changed
                 
-                _token           = value;
-                DbConfig.DbToken = value;
-                _isFirstInit     = true;
-            }
+            _token           = value;
+            DbConfig.DbToken = value;
+            _isFirstInit     = true;
         }
+    }
```

Comment:
Same as above

Reply:
Implemented in https://github.com/CollapseLauncher/Collapse/pull/791/commits/d9de2cac5a3ff89372844e29a4f27df6e7421f42 and https://github.com/CollapseLauncher/Collapse/pull/791/commits/1c1fdd3563b1e484c6fd880ff24f903e5eec044b